app/controllers/phone_numbers_controller.rb

```diff
@@ -56,7 +56,7 @@ def update
   def destroy
     @phone_number.destroy
     respond_to do |format|
-      format.html { redirect_to phone_numbers_url, notice: 'Phone number was successfully destroyed.' }
+      format.html { redirect_to @phone_number.person, notice: 'Phone number was successfully destroyed.' }
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.